animal-sniffer/src/main/java/org/codehaus/mojo/animal_sniffer/SignatureBuilder.java

```diff
@@ -199,8 +199,9 @@ protected void process( String name, InputStream image )
     {
         private Clazz clazz;
 
+        @SuppressWarnings("deprecation")
         public SignatureVisitor() {
-            super(Opcodes.ASM5);
+            super(Opcodes.ASM7_EXPERIMENTAL);
```

Comment:
`ASM7` now

Reply:
Done in 152ba6138f3e8f8b9745f21d7074aa79eea38624